runtime-modules/proposals/codex/src/proposal_types/mod.rs

```diff
@@ -47,6 +47,8 @@ pub enum ProposalDetails<
     /// The wasm code for the `runtime upgrade` proposal
     RuntimeUpgrade(Vec<u8>),
 
+    /// ********** Deprecated.
+    /// It is kept only for backward compatibility in the Pioneer. **********
```

Comment:
Why?

Reply:
Here is a good answer by Mokhtar: https://github.com/Joystream/joystream/pull/1510